modules/modtest/main.nf

```diff
@@ -0,0 +1,82 @@
+// Import generic module functions
+include { initOptions; saveFiles; getSoftwareName; getProcessName } from './functions'
+
+// TODO nf-core: If in doubt look at other nf-core/modules to see how we are doing things! :)
```

Comment:
Looks like you have accidentally tried to update 2 modules on this PR @sguizard ? The `modtest` one isn't finished? Could be worth creating separate PRs from scratch if it's easier.

Reply:
It's a dummy module I created to see the novelties of `versions.yaml`. 
I'm gonna delete this.

Reply:
Removed